test/missing_semicolon.html

```diff
@@ -21,4 +21,4 @@
 <body>
 
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
Right, sorry. I'm using atom which does this by default. I'll undo this and squash the commit.